noir-projects/aztec-nr/aztec/src/test/mocks/mock_note.nr

```diff
@@ -35,10 +34,19 @@ impl NullifiableNote for MockNote {
         )
     }
 
-    unconstrained fn compute_nullifier_without_context(self, storage_slot: Field) -> Field {
+    unconstrained fn compute_nullifier_without_context(
+        self,
+        storage_slot: Field,
+        contract_address: AztecAddress,
+        note_nonce: Field,
+    ) -> Field {
+        // We set the note_hash_counter to 0 as the note is not transient and the concept of transient note does
+        // not make sense in an unconstrained context.
+        let retrieved_note =
```

Comment:
It's not really that it doesn't make sense in unconstrained ctx (we can and do call this fn in the middle of private txs), but rather that this fn computes the nullifier for a note that is assumed to be comitted, it cannot compute transient nullifiers. This is what I alluded in a different comment as being odd. We should revisit this soon.

Reply:
Good point. Clarified this in https://github.com/AztecProtocol/aztec-packages/pull/11942/commits/19a786cbef010d1f0d4e6ac7bcd2acf5e1cad468